Tipstaff/Controllers/RespondentController.cs

```diff
@@ -73,19 +85,24 @@ public ActionResult Create(RespondentCreationModel model, string submitButton)
             try
             {
 
-                TipstaffRecord tr = db.TipstaffRecord.Find(model.tipstaffRecordID);
+                //////TipstaffRecord tr = db.TipstaffRecord.Find(model.tipstaffRecordID);
+                TipstaffRecord tr = _tipstaffRecordPresenter.GetTipStaffRecord(model.tipstaffRecordID);
```

Comment:
Is this necessary? we are doing it on line 56 (above). don't have to access DB again.

Reply:
But this is different Create method

Reply:
of course it is.